public/app/plugins/datasource/loki/configuration/DerivedField.tsx

```diff
@@ -97,8 +99,9 @@ export const DerivedField = (props: Props) => {
         />
       </div>
 
-      <div className={styles.row}>
+      <div className="gf-form">
```

Comment:
I think we could even remove on line 96 to have same spacing across whole form. 
```
css`
            margin-left: 8px;
          `
```

Reply:
Nice! Absolutely, removed in 9980f0e